modules/ocf_prometheus/manifests/alertmanager.pp

```diff
@@ -0,0 +1,40 @@
+class ocf_prometheus::alertmanager {
+  ocf::firewall::firewall46 {
+    '995 allow prometheus alertmanager to send on SMTP port':
+      opts   => {
+        chain       => 'PUPPET-OUTPUT',
+        proto       => 'tcp',
+        destination => ['anthrax'],
+        dport       => 25,
+        uid         => 'alertmanager',
```

Comment:
This depends on the `alertmanager` user as written.

Reply:
The `alertmanager` user is created by puppet-prometheus. Is it necessary to re-declare it here? I'd rather not write something that conflicts with what is written there.

Reply:
It should have a Requires

Reply:
Does it matter if this runs before the `alertmanager` user is created though? I don't think it would cause an error.

Reply:
I think it would, because there would be no UID to which to map that user.

Reply:
Fair point, I've added the requires.